extension/background/youtubei.js

```diff
@@ -54,3 +60,105 @@ function parseTimestamp(ts) {
     const hours = parseInt(parts[2]) || 0
     return secs + (60 * mins) + (60 * 60 * hours)
 }
+
+// time-comments related code below
+
+const INNERTUBE_API_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8"
+const INNERTUBE_CLIENT_NAME = "WEB"
+
+async function fetchTimeComments(videoResponse) {
+    const comments = await fetchComments(videoResponse)
+
+    // Currently using only the first minimally suitable comment.
+    // Maybe later implement more sophisticated comment selection.
+    for (let i = 0; i < comments.length; i++) {
+        const tsContexts = getTimestampContexts(comments[i].text)
```

Comment:
I'm afraid it's not that easy:
- What if there are multiple comments that have at least one "chapter"?
- What if one comment contains more chapters than another?
- What if chapters are in random (by time) order?
- May be we should prioritize a pinned comment?

Reply:
Okay, let's prioritize the pinned comment.